Makefile

```diff
@@ -2,16 +2,24 @@ SHELL := /bin/bash
 
 GO ?= $(shell which go)
 PROTOC ?= $(shell which protoc)
+MOCKGEN ?= $(shell which mockgen)
+
+ifeq ($(MOCKGEN),)
+# keep this in sync with the version of github.com/golang/mock in go.mod
+$(shell $(GO) get github.com/golang/mock/mockgen@v1.4.3) 
+endif
 
 TEST_FLAGS ?= -v -race
 
-PROTOS := pkg/api/schema/schema.proto pkg/api/output/output.proto pkg/api/version/version.proto
-PROTO_URLS := https://raw.githubusercontent.com/falcosecurity/falco/master/userspace/falco/schema.proto https://raw.githubusercontent.com/falcosecurity/falco/master/userspace/falco/output.proto https://raw.githubusercontent.com/falcosecurity/falco/master/userspace/falco/version.proto
-PROTO_SHAS := a1f427c114b945d0880b55058862b74015d036aa722985ca6e5474ab4ed19f69 4ce2f3e6d6ebc07a74535c4f21da73e44c6ef848ab83627b1ac987058be5ece9 0ac29f477e146a14504a34f91509668e44f4ee107d37b0be57e82847e8e89510
+PROTOS := pkg/api/schema/schema.proto pkg/api/outputs/outputs.proto pkg/api/version/version.proto
+PROTO_URLS := https://raw.githubusercontent.com/falcosecurity/falco/feat/bidi-grpc-outputs/userspace/falco/schema.proto https://raw.githubusercontent.com/falcosecurity/falco/feat/bidi-grpc-outputs/userspace/falco/outputs.proto https://raw.githubusercontent.com/falcosecurity/falco/feat/bidi-grpc-outputs/userspace/falco/version.proto
```

Comment:
todo: change this back to master before merging this once the corresponding PR gets merged into Falco https://github.com/falcosecurity/falco/pull/1241

Reply:
Done, about to commit!